container-host-files/etc/scf/config/scripts/patches/fix_route_registrar_bpm_dns.sh

```diff
@@ -17,6 +17,20 @@ if [ -f "${SENTINEL}" ]; then
   exit 0
 fi
 
+# Notes on "unsafe.unrestricted_volumes":
+#
+# - The mounts 1, 2, and 4 are required to make DNS work in the nested
+#   container created by BPM for the job to run in.
+#
+# - The remainer are required to give the job access to the system
+#   root certificates so that it actually can verify the certs given
+#   to it by its partners.
```

Comment:
This one has all of `/var/lib`, not just `/var/lib/ca-certificates`; is that intentional?

Reply:
Likely Jan simply missed it when he did the work ... I will make the fix now. And fix the typo I just saw.

Reply:
I also note leftover `/etc/pki` now. Will be removed.

Reply:
Done with commit 545bd60f8d80c04ed088d12aed2717db83377302